src/Sentry/Internal/Http/HttpTransport.cs

```diff
@@ -69,9 +67,9 @@ internal HttpRequestMessage CreateRequest(SentryEvent @event)
 
             var request = new HttpRequestMessage
             {
-                RequestUri = dsn.GetStoreEndpointUri(),
+                RequestUri = dsn.GetEnvelopeEndpointUri(),
                 Method = HttpMethod.Post,
-                Content = new StringContent(JsonSerializer.SerializeObject(@event))
+                Content = new StringContent(envelope.Serialize())
```

Comment:
Here we can go with `StreamContent` and get an envelope serialize itself into the stream.
Will be much better than getting a massive string allocated